lib/src/rules/avoid_positional_boolean_parameters.dart

```diff
@@ -60,8 +60,18 @@ class AvoidPositionalBooleanParameters extends LintRule
 class _Visitor extends SimpleAstVisitor<void> {
   final LintRule rule;
 
+  InheritanceManager manager;
+
   _Visitor(this.rule);
 
+  @override
+  void visitCompilationUnit(CompilationUnit node) {
+    LibraryElement library = node == null
```

Comment:
Visit methods should never be invoked with a null argument, so we shouldn't need this test. Did you run into an actual problem, or were you just being defensive?

Reply:
Literally just copied from `lib/src/rules/annotate_overrides.dart`. Good point. Deleting